internal/xds/rbac/matchers.go

```diff
@@ -307,29 +312,30 @@ func (upm *urlPathMatcher) match(data *rpcData) bool {
 	return upm.stringMatcher.Match(data.fullMethod)
 }
 
-// sourceIPMatcher and destinationIPMatcher both are matchers that match against
-// a CIDR Range. Two different matchers are needed as the source and ip address
-// come from different parts of the data about incoming RPC's passed in.
-// Matching a CIDR Range means to determine whether the IP Address falls within
-// the CIDR Range or not. They both implement the matcher interface.
-type sourceIPMatcher struct {
+// remoteIPMatcher and destinationIPMatcher both are matchers that match against
```

Comment:
Also rename `destinationIP` to `localIP`?

But the proto field is `v3rbacpb.Permission_DestinationIp`...
This inconsistency is annoying.

I would still rename `destinationIP` to `localIP`.
And also add a comment where `newDestinationIPMatcher` is called to say "Since this is on the server side, destination IP is the local IP"

Reply:
Another one is the `destinationAddr` field in `rpcData`.
That probably also makes sense since it's `destinationAddr: conn.LocalAddr()`

Reply:
Ok, but I'm only doing this because you make me laugh sometimes in our BS time.